doc/framework-integration/angular-instructions.md

```diff
@@ -66,37 +61,32 @@ import { AppComponent } from './app.component';
     InoElementsModule.forRoot(), // <-- b) make it available in Angular
   ],
   providers: [],
-  schemas: [CUSTOM_ELEMENTS_SCHEMA], // <-- c) Defines a schema that allows an NgModule to contain Non-Angular elements named with dash case (-) details https://angular.io/api/core/CUSTOM_ELEMENTS_SCHEMA
   bootstrap: [AppComponent],
 })
 export class AppModule {}
 ```
 
-2. Edit `main.ts`
-
-```typescript
-// src/main.ts
-
-import { enableProdMode } from '@angular/core';
-import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
-
-import { AppModule } from './app/app.module';
-import { environment } from './environments/environment';
-
-import { defineCustomElements } from '@inovex.de/elements/dist/loader';
+### 3) Use the Components
 
-if (environment.production) {
-  enableProdMode();
+### 4) (optional) Import the `ino-icon`s
+
+In order to use our icons, you need to add them to your build.
+To do so, add the following to the `assets` array under `projects.<your-project-name>.architect.build.options.assets` in your `angular.json` file:
+
+```json
+{
```

Comment:
The automatic solution would be to add a a schematic https://angular.io/guide/schematics-for-libraries#schematics-for-libraries and to add a command on `ng add` which copies the task to the angular.json (which is what ionic does). For now, we can leave it like this.

Reply:
> Looks good to me :) Here is one more contribution to remove ngcc and use partial ivy compiler for libraries
> => #387
> => see https://angular.io/guide/creating-libraries#ivy-libraries
> 
> Could you test if this works for you in James @Sl1nd ?

"Partial-Ivy code is not backward compatible with View Engine. If you use the library in a View Engine application, you must compile the library into the View Engine format by setting "enableIvy": false in the tsconfig.json file."

The "partial"  mode will lead to incompatibility issues with Angular v8 Projects using Elements. Shouldn't we warn our users and release the "partial" mode in the future? In the meantime i'll test James compatibility.

Reply:
> The automatic solution would be to add a a schematic https://angular.io/guide/schematics-for-libraries#schematics-for-libraries and to add a command on `ng add` which copies the task to the angular.json (which is what ionic does). For now, we can leave it like this.

Great Idea! Haven't thought of it. Did you create an Issue for this?

Reply:
see https://github.com/inovex/elements/issues/390